common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -535,6 +535,35 @@ def answer_problem(self, correctness):
         else:
             self.problem_page.click_choice("choice_choice_2")
 
+    @attr(shard=7)
+    def test_can_show_answer(self):
```

Comment:
The only pre-existing "show answer" test was for checkbox. That test was extended through the change in the page object (above) to check that show answer does inject the status HTML.

However, I also wanted to cover the "multiple checkmark" bug that I had seen in 2 different manifestations, and that is specific to multiple choice problems. Therefore I added a "show answer" test to this class too.